cypress/integration/test_stock_entry.js

```diff
@@ -0,0 +1,45 @@
+context('Create Stock Entry', () => {
+    before(() => {
+    cy.login();
+        });
+
+			//Set mode of payment first
+			it('Set Item Table', () => {
+			cy.visit('app/stock-entry');
+			cy.findByRole('button', {name: 'Add Stock Entry'}).trigger('click', {force: true});
+			cy.location("pathname").should("eq","/app/stock-entry/new-stock-entry-1");
+			cy.get_field('stock_entry_type', 'Link').type('Material Receipt');
+			cy.get_field('stock_entry_type', 'Link').should('have.value', 'Material Receipt');
+			cy.get('.rows > .grid-row > .data-row > .row-index').click();
```

Comment:
what does this do? Should be a separate cypress command.

Reply:
This clicks on a row area inside items table so as to expand it and open in full view

Reply:
This should definitely have a command if it's not already there!

`cy.expand_childtable_row(index)`